tests/unit/mixins/ui-service-mixin-test.js

```diff
@@ -0,0 +1,127 @@
+/* eslint-disable ember/no-mixins, ember/no-new-mixins, ember/no-classic-classes */
+
+import EmberObject from '@ember/object';
```

Comment:
suggestion: let's add the test to the first commit.

thought: I need some help walking through the test code - I'll schedule something for this afternoon.